projects/ngx-prx-styleguide/src/lib/select/select.component.ts

```diff
@@ -1,5 +1,5 @@
-import { Component, Input, Output, EventEmitter, OnChanges, SimpleChanges } from '@angular/core';
-import { NgOption, NgSelectConfig } from '@ng-select/ng-select'
+import { Component, Input, Output, EventEmitter, OnChanges, SimpleChanges, ViewChild } from '@angular/core';
```

Comment:
What's the change in imports for?

Reply:
(Maybe don't need that `ViewChild`)

Reply:
Related to my failed attempt to use templates for this. Removed.